librz/il/internal_ct_cmd.c

```diff
@@ -0,0 +1,480 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
```

Comment:
I am not sure if `internal_ct_cmd.c` is a good name for this file. It looks more like `librz/il/vm.c` instead. Same for corresponding header.

Reply:
Yep this one is not `internal` anymore, what do you think of `vm_layer.c` ?

Reply:
Yes, sounds good to me.